platform-sdk/docs/core/dynamic-address-book/roster/glossary-of-terms.md

```diff
@@ -0,0 +1,100 @@
+# Roster Glossary of Terms
+![Event Window Terminology](./EventWindowTerminology.drawio.svg)
+![Roster Terminology Diagram](./RosterTerminologyDiagram.drawio.svg)
+## Consensus Terminology
+
+### Latest Consensus Round
+
+The most recent round to have reached consensus.  (equal to the pending consensus round - 1)
+
+### Pending Consensus Round
+
+The round that the consensus algorithm is currently attempting to reach consensus on. (equal to the latest 
+consensus round + 1)
+
+### Rounds Non-Ancient
+
+A setting. The number of rounds that have reached consensus that are not ancient.  Currently set to 26 in all 
+environments. 
+
+### Ancient
+
+An event is ancient if its `birthRound` is less than or equal to the`latest consensus round` - `rounds non-ancient`.
+
+### Minimum Round Non-Ancient
+
+`minRoundNonAncient` is the minimum round that is considered non-ancient.  This is equal to the `pending consensus 
+round` - `rounds non-ancient`.
+
+### Non-Ancient Event Window
+
+Specifies the window of events that are being considered in consensus.  The window specified the following:
+* `latestConsensusRound` - the latest round that has reached consensus.
+* `minRoundNonAncient` - the minimum round that is considered non-ancient.
+* `minGenNonAncient` - the minimum generation that is considered non-ancient.  (This is deprecated). 
+
+## Roster Terminology
+
+### Roster (a.k.a Consensus Roster)
+
+The collection of nodes which create events and participate in consensus.  Prior to the dynamic address book effort, 
+this was called the `AddressBook` in the platform's code.  The name was changed to `Roster` to disambiguate it from 
+Hedera's concept of an address book.  Hedera's address book contains the consensus roster.  
```

Comment:
```suggestion
the application's concept of an address book.  The application's address book contains the consensus roster.  
```

Reply:
all mention of hedera are removed from my local branch.